UKHomeOffice/cerberus-service
# COP-10671 : Use feature STANDARDISED:arrivalPort/departurePort fields

## Description
COP-10671: Use feature STANDARDISED:arrivalPort/departurePort fields if present

https://support.cop.homeoffice.gov.uk/browse/COP-10671

All scenarios Covered
## To Test
Run the tests from spec create-airpax-tasks.spec.js
```
Should create an airpax task with STANDARDISED:arrivalPort/departurePort fields present
Should create an airpax task with STANDARDISED:arrivalPort/departurePort fields not present
```
## Developer Checklist

\* Required
